### DISCUSSION TITLE
Project Manager should correct invalid paths

### DISCUSSION THREAD

**Discussion by Ryder17z** [godotengine/godot-proposals#6528] [General Discussions]
[Example scenario](https://user-images.githubusercontent.com/2000703/226110144-dd9835a4-79c1-48c8-b7c0-2db96176fd1a.png)

On windows, a slash in the name of a folder is for example not allowed, this could be easily solved by having the script responsible for creating the folder replace the invalid characters by a dash (-) or an underscore (_) for example.

**Comment by Calinou** [comment]
Feel free to open a pull request for this :slightly_smiling_face: 

The responsible code is somewhere around https://github.com/godotengine/godot/blob/a7d0e18a317085068c43be29bca1d280d03423a2/editor/project_manager.cpp#L805.
